src/index.ts

```diff
@@ -28,7 +28,7 @@ type PluginOptions = {
 export default function vitePluginCreateLitReactWrapper(
   {
     globToLitComponents,
-    componentPrefix,
+    prefix,
```

Comment:
Since we are using this for events and components I generalized the name to just "prefix"